airgun/views/file.py

```diff
@@ -0,0 +1,54 @@
+from widgetastic.widget import Text, View
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    ReadOnlyEntry,
+    SatTab,
+    SatTable,
+)
+from airgun.widgets import Search
+
+
+class FilesView(BaseLoggedInView):
+    """Main Files view"""
+
+    title = Text("//h2[contains(., 'Files')]")
```

Comment:
As I am looking at the Files page the Title is h1 and not h2. Update the locator please.
```suggestion
    title = Text("//h1[contains(., 'Files')]")
```